src/Instrumentation/Laravel/tests/Integration/ConsoleInstrumentationTest.php

```diff
@@ -4,43 +4,11 @@
 
 namespace OpenTelemetry\Tests\Contrib\Instrumentation\Laravel\Integration;
 
-use ArrayObject;
 use Illuminate\Console\Command;
 use Illuminate\Contracts\Console\Kernel;
-use OpenTelemetry\API\Instrumentation\Configurator;
-use OpenTelemetry\Context\ScopeInterface;
-use OpenTelemetry\SDK\Trace\ImmutableSpan;
-use OpenTelemetry\SDK\Trace\SpanExporter\InMemoryExporter;
-use OpenTelemetry\SDK\Trace\SpanProcessor\SimpleSpanProcessor;
-use OpenTelemetry\SDK\Trace\TracerProvider;
 
 class ConsoleInstrumentationTest extends TestCase
 {
-    private ScopeInterface $scope;
-    private ArrayObject $storage;
-
-    public function setUp(): void
-    {
-        parent::setUp();
-
-        $this->storage = new ArrayObject();
-        $tracerProvider = new TracerProvider(
-            new SimpleSpanProcessor(
-                new InMemoryExporter($this->storage)
-            )
-        );
-
-        $this->scope = Configurator::create()
-            ->withTracerProvider($tracerProvider)
-            ->activate();
-    }
-
-    public function tearDown(): void
-    {
-        $this->scope->detach();
-        parent::tearDown();
-    }
-
```

Comment:
Provided by `WithInstrumentation` trait.

Reply:
how does it work? I see `WithInstrumentation` provides methods like s`setUpWithInstrumentation` rather than the `setUp` that phpunit looks for.

Reply:
In Laravel, this kind of trait setup is pretty common. Testbench provides this functionality [here](https://github.com/orchestral/testbench-core/blob/11d12200314f86e4c05e719c631fedc94a68a20c/src/Concerns/Testing.php#L227-L249).

You can simplify the test `setUp` a little where you are using many traits. I can change this back to vanilla `setUp`/`tearDown` if preferred :angel:

Reply:
As per today's PHP-SIG discussion, this has now been moved to the standard `setUp`/`tearDown` in the base TestCase.